apps/web/src/components/UpcomingLessons/Content.tsx

```diff
@@ -104,7 +104,7 @@ export const Content: React.FC<{
   if (!list || !list.length || !user)
     return (
       <div className="mt-[15vh]">
-        <EmptyLessons tutorsPage={Web.Tutors} />;
+        <EmptyLessons tutorsPage={Web.Tutors} isStudent={user?.role === IUser.Role.Student} />;
```

Comment:
You can make use of the new util function `isStudent(user)` and it will work. https://github.com/litespace-org/litespace/blob/4434f0bb9f04a573f9887d23e26eb7c092ddb33b/packages/utils/src/user.ts#L61-L63